packages/cli/src/swagger/specGenerator3.ts

```diff
@@ -5,6 +5,19 @@ import { convertColonPathParams, normalisePath } from './../utils/pathUtils';
 import { SpecGenerator } from './specGenerator';
 import { UnspecifiedObject } from '../utils/unspecifiedObject';
 
+function getValue(type: 'string' | 'number' | 'integer' | 'boolean', member: any) {
```

Comment:
Handle null

Reply:
Isn't `null` handled implicitly already? -> The `getValue` cannot be called with `null`.